README.md

```diff
@@ -1,108 +1,116 @@
-
-
-### Connect the database
-Here we used the mysql database. That database was created in a machine and linked to the other machines.
-Original machine had the database and it gave the permission to the other machine through this command
-       
-        Mysql -u root -p;
-        ALTER USER ‘username’@’%’ IDENTIFIED BY ‘password’;
-        Flush privileges;
-        grant all on *.* to 'root'@'%';
-        flush privileges;
-The client accessed through the 
-        
-        mysql -h <IP_ADDRESS_OF_REMOTE_HOST> -u <USERNAME> -p
-        Eg: mysql -h 10.100.4.205 -u root -p
-
-Here the client also wanted to know the password of the mysql of the original.
-
-
+# WSO2 API Manager Ansible scripts
+
+This repository contains the Ansible scripts for installing and configuring WSO2 API Manager.
+
+## Supported Operating Systems
+
+- Ubuntu 16.04 or higher
+
+## Supported Ansible Versions
+
+- Ansible 2.0.0.2
+
+## Directory Structure
+```
+.
+├── dev
+│   ├── group_vars
+│   │   └── am.yml
+│   ├── host_vars
+│   │   ├── am_1.yml
+│   │   └── am_2.yml
+│   └── inventory
+├── docs
+│   ├── images
+│   │   ├── API-M-single-node-deployment.png
+│   │   ├── P-H-1.png
+│   │   ├── P-H-2.png
+│   │   ├── P-H-3.png
+│   │   ├── P-M-1.png
+│   │   └── P-S-1.png
+│   ├── Pattern_1.md
+│   ├── Pattern_2.md
+│   ├── Pattern_3.md
+│   ├── Pattern_4.md
+│   └── Pattern_5.md
+├── files
+│   ├── mysql-connector-java-5.1.45-bin.jar
+│   └── wso2am-linux-installer-x64-2.5.0.deb
+├── issue_template.md
+├── LICENSE
+├── pull_request_template.md
+├── README.md
+├── roles
+│   └── am
+│       ├── tasks
+│       │   ├── custom.yml
+│       │   └── main.yml
+│       └── templates
+│           ├── carbon-home
+│           │   ├── bin
+│           │   │   └── wso2server.sh.j2
+│           │   └── repository
+│           │       └── conf
+│           │           ├── api-manager.xml.j2
+│           │           ├── axis2
+│           │           │   └── axis2.xml.j2
+│           │           ├── carbon.xml.j2
+│           │           ├── datasources
+│           │           │   └── master-datasources.xml.j2
+│           │           ├── identity
+│           │           │   └── identity.xml.j2
+│           │           ├── registry.xml.j2
+│           │           ├── tomcat
+│           │           │   └── catalina-server.xml.j2
+│           │           └── user-mgt.xml.j2
+│           └── wso2am.service.j2
+└── site.yml
+
+```
+
+## Packs to be Copied
+
+Copy the following files to `files` directory.
+
+1. [WSO2 API Manager 2.5.0 package](https://wso2.com/api-management/install/)
+2. [mysql-connector-java-5.1.45-bin.jar](https://dev.mysql.com/downloads/connector/j/5.1.html)
```

Comment:
@VimukthiPerera the correct link for downloading the defined, archived version of MySQL Connector/J, can be found from [here](https://downloads.mysql.com/archives/c-j/).

At present, the latest version is 5.1.47. Hence, we can either state this link as `[MySQL Connector/J](https://dev.mysql.com/downloads/connector/j/5.1.html)` in the README documentation, in order to avoid mentioning the version or we can provide the above mentioned archived version link.